packages/ecommerce-app-base/src/Editor/SortableListItem.tsx

```diff
@@ -125,17 +124,24 @@ export const SortableListItem = SortableElement<Props>(
     const productIsMissing = !product.name;
 
     return (
-      <Card className={styles.card}>
+      <Card
+        data-testid="SortableListItem"
```

Comment:
nit: sometimes it's pascal case and sometimes kebab. Worth unifying?

Reply:
I use pascal case when the property is on the top element of a component. That way the test id basically references the full component.

Gonna make it consistent anyway